config/config.sample.edn

```diff
@@ -66,4 +66,5 @@
   :imgflip {:username "", :password ""},
   :admin {:commands ["observer" "obs"]
           :users ["U123123" "~awesomeperson"]},
-  :nrepl {:port ""}}}
+  :nrepl {:port ""}
+  :karma {:emoji {:positive "🌮" :negative "💩"}}}}
```

Comment:
I know it's a pain but can you add the samples to the alternative config format in `config/profiles.sample.clj`? In the future we'll autogen these after we migrate to spec.